core/Files.h

```diff
@@ -100,6 +101,9 @@ class File final {
     struct Flags {
         // some reasonable invariants don't hold for invalid files
         bool hasIndexErrors : 1;
+        // only relevant in --stripe-packages mode: is the file contained in a `/test/` directory and not a `test.rb`
+        // file?
+        bool isPackagedTestHelper : 1;
         // only relevant in --stripe-packages mode: is the file a `.test.rb` file?
```

Comment:
I think this comment is incorrect actually. It might be worth changing it eventually to be just `.test.rb` files, but that's a bigger refactor we can do later.
```suggestion
        // only relevant in --stripe-packages mode: is the file contained in a `/test/` directory or a `.test.rb` file?
```